src/index.js

```diff
@@ -1,3 +1,4 @@
+/*eslint-disable */
```

Comment:
- Please, kindly fix the eslint error instead of disabling the linter as this is pretty easy to fix. kindly use your spacebar to add two indentations and add the missing comma. see the image below:

![image](https://github.com/hmunish/math-magicians/assets/59397159/ec6fce57-940c-483b-89e6-facf9f71fa53)